commands/operator-sdk/cmd/test/cluster.go

```diff
@@ -48,13 +47,8 @@ func NewTestClusterCmd() *cobra.Command {
 		Short: "Run End-To-End tests using image with embedded test binary",
 		RunE:  testClusterFunc,
 	}
-	defaultKubeConfig := ""
-	homedir, ok := os.LookupEnv("HOME")
-	if ok {
-		defaultKubeConfig = homedir + "/.kube/config"
-	}
-	testCmd.Flags().StringVar(&tcConfig.namespace, "namespace", "default", "Namespace to run tests in")
-	testCmd.Flags().StringVar(&tcConfig.kubeconfig, "kubeconfig", defaultKubeConfig, "Kubeconfig path")
+	testCmd.Flags().StringVar(&tcConfig.namespace, "namespace", "", "Namespace to run tests in")
+	testCmd.Flags().StringVar(&tcConfig.kubeconfig, "kubeconfig", "", "Kubeconfig path")
```

Comment:
Are you going to allow someone to pass a context with the kubeconfig? 

If we want to tackle it later makes sense just a thought.

Reply:
It looks like we could pass a context through the `config.Overrides` that are passed to `NewDefaultClientConfig` (speaking of which, I just realized I can move those lines out of the if statement, since that line is identical in both the if and else; I'll fix that before merging). I assume this question is related to #351. I didn't add support in `up local` here, which is what #351 needs, but we can definitely add support for that.